model/src/ModelCommandAPI_TEST.cc

```diff
@@ -0,0 +1,333 @@
+/*
+ * Copyright (C) 2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+#include "cmd/ModelCommandAPI.hh"
+
+#include <cstdio>
+#include <cstdlib>
+#include <string>
+
+#include <gtest/gtest.h>
+
+#include "ignition/gazebo/Server.hh"
+
+static const std::string kIgnModelCommand(std::string(IGN_PATH) +
+                                          "/ign model ");
+
+
+/////////////////////////////////////////////////
+// Used to avoid the cases where the zero is represented as a negative number.
+std::string ReplaceNegativeZeroValues(std::string s)
```

Comment:
1. argument can be `const &` 
2. match style in the argument name

Reply:
Addressed